xbmc/video/ContextMenus.cpp

```diff
@@ -276,11 +185,17 @@ void SetPathAndPlay(CFileItem& item)
   }
   else
   {
+    // auto play next item only works from inside certain media windows
+    VIDEO_UTILS::PlayMode mode = VIDEO_UTILS::PlayMode::PLAY_ONLY_THIS;
+    const int activeWindow = CServiceBroker::GetGUI()->GetWindowManager().GetActiveWindow();
```

Comment:
same here it'd be nice to avoid checking the active window and just use a property of the item

Reply:
Also, any reason to have `VIDEO_UTILS::PlayMode::` and `MUSIC_UTILS::PlayMode::` and not only a single one (like content utils?

Reply:
Do  you have an idea where exactly the common definitions could be put, which header file I mean?

Reply:
ContentUtils.h?

Reply:
Okay.